tests/rbd_mirror/test_rbd_mirror_peer_id.py

```diff
@@ -0,0 +1,212 @@
+    """
+    try:
+        log.info("Starting snapshot RBD mirroring test case")
+        config = deepcopy(kw.get("config").get(pool_type))
+
+        for pool_name, pool_config in getdict(config).items():
+            multi_image_config = getdict(pool_config)
+
+            for image, image_config in multi_image_config.items():
+                # Verify that "rbd mirror pool status" reports "health: OK"on primary cluster
+                if check_pool_mirror_status(mirror1, pool_name, "Primary_site", "OK"):
+                    return 1
+                log.info("Primary site health is OK")
+
+                # Verify that "rbd mirror pool status" reports "health: OK"on secondary cluster
+                if check_pool_mirror_status(mirror2, pool_name, "Secondary_site", "OK"):
+                    return 1
+                log.info("Secondary site health is OK")
+
+                # Grab service_id and instance_id from secondary cluster
+                pool_config = {"pool": pool_name, "verbose": True, "format": "json"}
+                output, err = mirror2.mirror.pool.status(**pool_config)
+                pool_status = json.loads(output)
+                service_id = pool_status["daemons"][0]["service_id"]
+                instance_id = pool_status["daemons"][0]["instance_id"]
+
+                log.info(f"Service ID: {service_id}")
+                log.info(f"Instance ID: {instance_id}")
+
+                # Grab peer UUID from secondary
+                output, err = mirror2.mirror.pool.info(pool=pool_name, format="json")
+                output_json = json.loads(output)
+                peer_uuid = output_json["peers"][0]["uuid"]
+                log.info(f"Peer ID of secondary cluster: {peer_uuid}")
+
+                # Set invalid peer client id on secondary
+                mirror2.mirror.pool.peer.set_(
+                    pool=pool_name,
+                    uuid=peer_uuid,
+                    key="client",
+                    value="client.invalid",
+                )
+
+                # Verify that "rbd mirror pool status" reports "health: ERROR" on secondary
+                if check_pool_mirror_status(
+                    mirror2, pool_name, "Secondary_site", "ERROR"
+                ):
+                    return 1
+                log.info("Secondary site health is ERROR as expected")
+
+                # Verify that "rbd mirror pool status" reports "health: WARNING" on primary
+                if check_pool_mirror_status(
+                    mirror1, pool_name, "Primary_site", "WARNING"
+                ):
+                    return 1
+                log.info("Primary site health is WARNING as expected")
+
+                # Reset valid peer client id on secondary i.e client.rbd-mirror-peer
+                mirror2.mirror.pool.peer.set_(
+                    pool=pool_name,
+                    uuid=peer_uuid,
+                    key="client",
+                    value="client.rbd-mirror-peer",
```

Comment:
You are supposed to set it with the value retrieved previously before setting invalid right? Why are you setting it with a hardcoded value "client.rbd-mirror-peer"

Reply:
As discussed implemented that way from `rbd mirror pool info` ouput